documentation/source/building.rst

```diff
@@ -62,6 +62,17 @@ Make Variables
 
       Passed to both the compile and execute phases of simulators with two rules, or passed to the single compile and run command for simulators which don't have a distinct compilation stage.
 
+.. make:var:: PLUSARGS
+
+      "Plusargs" are options that are starting with a plus (``+``) sign. 
+      They are passed to the simulator and are also available within cocotb as :attr:`cocotb.plusargs`.
+      In the simulator, they can be read by the (System)Verilog system functions 
+      ``$test$plusargs`` and ``$value$plusargs``.
+
+      The special plusargs ``+ntb_random_seed`` and ``+seed``, if present, are evaluated
```

Comment:
Is this simulator specific, or handled by cocotb internally?

Reply:
It's handled by cocotb in `__init__.py`. IIRC, `ntb_random_seed` is actually a VCS thing, Mentor and Cadence use `svseed` instead. 

I'd be happy to deprecate this special handling, it seems rather hard to discover and doesn't work universally. But let's discuss that in a separate issue, documenting the current behavior is the right thing to do for this PR.

Reply:
Opened #1252 for that.